Arrowgene.Ddon.Server/GameLogicSetting.cs

```diff
@@ -17,16 +17,25 @@ public class GameLogicSetting
         [DataMember(Order = 1)]
         public double AdditionalCostPerformanceFactor { get; set; }
 
+        /// <summary>
+        /// Controls whether to use the possibly expensive lobby handling.
+        /// True = packet-intensive on entering a lobby zone (WDT, Megado, Breya Coast event space, etc.), but addresses one-sided invisibility.
+        /// </summary>
+        [DataMember(Order = 2)]
+        public bool FancyLobbyContextHandling { get; set; }
```

Comment:
Maybe we can make a better name than fancy context handling? ;)

Reply:
I'd invert the name, i'd have the flag to enable the, say, naive context handling? any other name for the more complex context handling like "fancy" or "new" will be outdated if we ever figure out a better way of handling and we want a feature flag for its replacement